tools/amd_build/pyHIPIFY/rccl1_compat.h

```diff
@@ -0,0 +1,53 @@
+/*************************************************************************
+* Copyright (c) 2017, AMD. All rights reserved.
```

Comment:
This copyright will not be OK for upstream. Can we get that changed?

Reply:
Yes, will remove it.

Reply:
Check w/ whoever owns this thing first.